test/e2e/pages/allocation-criteria.js

```diff
@@ -0,0 +1,53 @@
+import { Selector } from 'testcafe'
+
+import { fillInForm } from '../_helpers'
+
+import Page from './page'
+
+class AllocationCriteria extends Page {
```

Comment:
Can we continue to use the consistent naming of pages and include `Page` in the name: `AllocationCriteriaPage`.